source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
featured imaze size should be 1024 x 600 size atleast, as other blog is having there in nayan blog.

Reply:
Done

Reply:
Good